src/routes/index.js

```diff
@@ -1,22 +1,70 @@
 const express = require('express')
+const jwt = require('jsonwebtoken')
+const User = require('../models/user')
+const { makePassword } = require('../utils')
+const { getLatestMessages } = require('../controllers')
+// Loading .env
+require('dotenv').config()
+
 const router = express.Router()
-const Message = require('../models/message')
 
-router.get('/login', (req, res) => {
-  const room = {
-    website: req.query.website,
-    hateSpeechFlag: false,
+router.post('/login', async (req, res) => {
+  const user = await User.findOne({
```

Comment:
Move the business logic of this route to a separate file, and break it into separate functions.